src/Symfony/Component/Notifier/Bridge/Nexmo/README.md

```diff
@@ -1,7 +1,20 @@
 Nexmo Notifier
 ==============
 
-Provides Nexmo integration for Symfony Notifier.
+Provides [Nexmo](https://nexmo.com) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```ini
+# .env file
+nexmo://KEY:SECRET@default?from=FROM
```

Comment:
The DSN should be assigned to some env variable, shouldn't it?

Reply:
hmm yes you are right, for now we just keep it like this, maybe its better to remove the `# .env file` part, because it just show the DSN itself.

Reply:
Check https://github.com/symfony/symfony/pull/39383/commits/dd1f82a845805def8d8b0cbaa3b57279627b8f5b